### DISCUSSION TITLE
Verona preview and documentation seems down

### DISCUSSION THREAD

**Discussion by hisem** [primefaces/community#1468] [PrimeVue Templates] [answered]
I have been getting the following when trying to access https://verona.primevue.org/ since yesterday:

```
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
```

Is the template's website down?

**Comment by cagataycivici** [comment]
Thank you for the heads up, let me check.

**Reply by DuckyProject** [reply]
It is down for one month. Please fix that please.

**Comment by kelvinmarlow** [comment]
> I have been getting the following when trying to access https://verona.primevue.org/ since yesterday:
> 
> ```
> 404: NOT_FOUND
> Code: DEPLOYMENT_NOT_FOUND
> ```
> 
> Is the template's website down?

it`s down here yet

**Comment by jiliop** [comment]
Hi @cagataycivici . My dev team wants to buy Verona template but unfortunately we are not sure if it is safe to buy it, because the "preview" is not working. 
Is currently Verona a working template, or should we choose another?

Thank you in advance.

**Comment by hisem** [comment, marked as answer]
I see that the preview website is now back up 👍
